src/app/constants.ts

```diff
@@ -0,0 +1,38 @@
+export const REQUIRED: string = $localize`:@@required:Required`;
+export const DOWNLOAD: string = $localize`:@@download:Download`;
+export const LOGIN: string = $localize`:@@login:Login`;
+export const NEXT: string = $localize`:@@logout:Suivant`;
+export const PREVIOUS: string = $localize`:@@previous:Retour`;
+export const WRONG_EMAIL: string = $localize`:@@wrong_email:Format de courriel incorrect`;
```

Comment:
Are we moving all these constants from some other file?
Or why do we have many new constants in this context?